cmd/formatter/colors.go

```diff
@@ -86,38 +86,31 @@ func makeColorFunc(code string) colorFunc {
 }
 
 var nextColor = rainbowColor
+var rainbow []colorFunc
+var currentIndex = 0
 
 func rainbowColor() colorFunc {
-	return <-loop
+	result := rainbow[currentIndex]
```

Comment:
this isn't thread safe, would require a mutex (this is why we used a channel here)